### PR TITLE
Merge pull request #840 from minrk/py38

### PR DESCRIPTION
**Pull request by minrk**
includes #839, needed to get versions that support Python 3.8

### CODE REVIEW

File: repo2docker/buildpacks/conda/freeze.py [hunk 1]
```diff
@@ -76,7 +76,7 @@ def freeze(env_file, frozen_file):
             "; ".join(
                 [
                     "set -ex",
-                    f"conda install -yq conda={CONDA_VERSION}",
+                    f"conda install -yq -S conda={CONDA_VERSION}",
```
Comment (betatim) [inline repo2docker/buildpacks/conda/freeze.py]: What does the `-S` do?

**Comment by betatim** [comment]
This looks good to me. Merging.

Thanks for the maintenance'ing!

**Comment by betatim** [comment]
I am confused. Did I just click merge in this PR and #839? I didn't think so but now both are merged :-/

**Comment by consideRatio** [comment]
@betatim I think GitHub realized that the commits within #839 was merged (with exact sha etc), so then it was automatically considered to be merged also.

**Comment by consideRatio** [comment]
From within #839, it said:

> @betatim merged commit 2e2d1f4 into jupyter:master 4 hours ago

If you look in the list of commits in this PR, this was one of the commits.

**Comment by betatim** [comment]
Black magic!

**Pull request merged**